crates/bevy_asset/src/io/embedded/mod.rs

```diff
@@ -135,12 +135,12 @@ macro_rules! embedded_path {
 ///     * `src`
 ///         * `render`
 ///             * `rock.wgsl`
-///             * `mod.rs`
+///             * `gpu_resource`
```

Comment:
What is this change for? It replaced a clear filename with something entirely different.